src/main.rs

```diff
@@ -60,7 +71,17 @@ fn build_subcommands<'a, 'b>(
         }
         cli_app = cli_app.subcommand(subcmd);
     }
-    cli_app
+
+    // This is needed to prevent clap from complaining. It should be removed once
+    // clap 3.x is released. See https://github.com/clap-rs/clap/issues/748
+    let maskfile = Arg::with_name("maskfile")
```

Comment:
Just a minor nitpick 👍 
```suggestion
    let custom_maskfile_path = Arg::with_name("maskfile")
```